gatsby-config.js

```diff
@@ -163,7 +163,9 @@ module.exports = {
       resolve: `gatsby-plugin-manifest`,
       options: {
         name: 'React Docs',
-        short_name: 'React',
+        short_name: 'React [%LANG_CODE%]', // eg. React [%LANG_CODE%]
```

Comment:
@lex111 is that correct? or it has to be like this:
```suggestion
        short_name: 'React pt-br', // eg. React [%LANG_CODE%]
```

Reply:
@jhonmike this is at your discretion, just a short name to distinguish between the original documentation and translation, I think the brackets can be added for beauty:

```suggestion
        short_name: 'React [pt-br]', // eg. React [%LANG_CODE%]
```